src/roman_numerals/core.clj

```diff
@@ -1,7 +1,12 @@
 (ns roman-numerals.core)
 
-(defn convert
-  "I convert arabic to roman numerals"
-  [x]
-  (if (= x 0)
-    ""))
+(defn to-numerals [arabic numerals] 
+  (cond (= arabic 0) numerals
+        (<= arabic 3) (to-numerals (- arabic 1) (str numerals "I"))
+        (= arabic 4)  (to-numerals (- arabic 4) (str numerals "IV"))
+        (= arabic 9) (to-numerals (- arabic 9) (str numerals "IX"))
+        (>= arabic 10) (to-numerals (- arabic 10) (str numerals "X"))
+        (>= arabic 5) (to-numerals (- arabic 5) (str numerals "V"))))
```

Comment:
this is a nitpick, but I would order the implementation in the descending order of the numbers , since the order matters here